index.js

```diff
@@ -74,4 +72,4 @@ for( var l in levels ) { ( function( prio, name ) {
 		log( prio, message, fields );
 	};
 
-} )( l, levels[ l ] ); }
+} )( l * 10, levels[ l ] ); }
```

Comment:
Why do you multiply the level by 10 for the communication between JS and the C++ stuff?